app/controllers/attachments_controller.rb

```diff
@@ -0,0 +1,6 @@
+class AttachmentsController < ApplicationController
```

Comment:
Sure. I called it Attachments because that's what the route is called, and I figured that inline attachments would start to be listed here.

Reply:
Gotcha - my understanding is that we intend for this controller to only be used for functionality related to featured attachments - which probably makes sense since the convergence of these things is super confusing.